src/test/kotlin/com/coder/gateway/settings/CoderSettingsTest.kt

```diff
@@ -170,6 +170,16 @@ internal class CoderSettingsTest {
         assertEquals(Pair("http://test.gateway.coder.com$expected", "fake-token"), got)
     }
 
+    @Test
+    fun testSSHConfigOptions() {
+        var settings = CoderSettings(CoderSettingsState(sshConfigOptions = "ssh config options from state"))
+        assertEquals("ssh config options from state", settings.sshConfigOptions)
+
+        settings = CoderSettings(CoderSettingsState(),
+            env = Environment(mapOf(CODER_SSH_CONFIG_OPTIONS to "ssh config options from env")))
+        assertEquals("ssh config options from env", settings.sshConfigOptions)
+    }
```

Comment:
What's the precedence here? State or env?

Reply:
State, and good point I will test that!